backend/database/migrations/2022_03_11_023741_change_date_as_nullable_to_attempts_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('attempts', function (Blueprint $table) {
+            DB::statement("ALTER TABLE `attempts` MODIFY `date_started` TIMESTAMP NULL");
+            DB::statement("ALTER TABLE `attempts` MODIFY `date_finished` TIMESTAMP NULL");
+        });
+    }
+
```

Comment:
This works but not advisable, 

Please use this instead using the DB statement
https://laravel.com/docs/7.x/migrations#modifying-columns